lms/djangoapps/verify_student/views.py

```diff
@@ -1047,3 +1049,82 @@ def reverification_window_expired(_request):
     """
     # TODO need someone to review the copy for this template
     return render_to_response("verify_student/reverification_window_expired.html")
+
+
+class ICRVReverifyView(View):
+    """
+    The in-course reverification view.
+    Needs to perform these functions:
+        - take new face photo
+        - retrieve the old id photo
+        - submit these photos to photo verification service
+
+    Does not need to worry about pricing
+    """
+    @method_decorator(login_required)
+    def get(self, request, course_id, checkpoint_name):
+        """
+        display this view
+        """
+        user = request.user
+        course_id = CourseKey.from_string(course_id)
+        course = modulestore().get_course(course_id)
+        if course is None:
+            raise Http404
+        checkpoint = VerificationCheckpoint.get_verification_checkpoint(course_id, checkpoint_name)
+        if checkpoint is None:
+            raise Http404
+        try:
+            SoftwareSecurePhotoVerification.original_verification(user)
+        except IndexError:
+            path = reverse('verify_student_verify_later', kwargs={'course_id': unicode(course_id)})
+            return redirect(path)
+        context = {
+            "user_full_name": request.user.profile.name,
+            "error": False,
+            "course_id": course_id.to_deprecated_string(),
+            "course_name": course.display_name_with_default,
+            "course_org": course.display_org_with_default,
+            "course_num": course.display_number_with_default,
+            "checkpoint_name": checkpoint_name,
+        }
+
+        return render_to_response("verify_student/midcourse_photo_reverification.html", context)
```

Comment:
We don't want to re-use this template.  Among other things, it doesn't support iOS photo capture and uses some really messy JavaScript.

For this story, please create a new template based on `pay_and_verify.html` and re-use the existing Backbone views/models to collect and submit the webcam photo.